### PR TITLE
chore(react-tree-grid): adds playwright component tests

### PR DESCRIPTION
**Pull request by bsunderhus**
1. adds playwright component test configuration
2. adds playwright test for TreeGrid keyboard/click behaviours

### CODE REVIEW

**Comment by Hotell** [comment]
once this lands to main https://github.com/microsoft/fluentui-contrib/pull/102

- re-generate pw config

**Comment by Hotell** [comment]
we need to discuss file name patterns, because with current setup if project would use jest and pw component test, the only distinction ATM is `.test` vs `.spec` but jest actually invokes also `.spec` which would explode.

with cypress we use `*.cy.*` suffix, maybe we can do similar for playwright, something like `.pw.tsx` etc

**Comment by bsunderhus** [comment]
> once this lands to main #102
> 
> * re-generate pw config

@Hotell,

Trying to follow up on this, I'm getting some errors while trying to test locally

image

I went to `main` branch and regenerated the playwright configuration and  a spec test, and this happens whenever I try to run `component-test`

**Comment by Hotell** [comment]
ups, while in rush I created invalid options generation dictionary 

fix https://github.com/microsoft/fluentui-contrib/pull/116

File: packages/react-tree-grid/jest.config.ts [hunk 1]
```diff
@@ -18,12 +19,15 @@ if (swcJestConfig.swcrc === undefined) {
...
-export default {
+const config: Config = {
   displayName: 'button',
   preset: '../../jest.preset.js',
   transform: {
     '^.+\\.[tj]sx?$': ['@swc/jest', swcJestConfig],
   },
   moduleFileExtensions: ['js', 'ts', 'tsx', 'html'],
   coverageDirectory: '../../coverage/packages/button',
+  modulePathIgnorePatterns: ['.spec.tsx'],
```
Comment (Hotell) [inline packages/react-tree-grid/jest.config.ts:30]: this will hit us sooner than later. I'll create issue to track this

Comment (Hotell) [reply packages/react-tree-grid/jest.config.ts:30]: https://github.com/microsoft/fluentui-contrib/issues/118

**Review by Hotell — APPROVED** [review]

**Pull request merged**